File: test/fixtures/code/sample.go
package sample

import "strings"

type SampleService struct {
	Prefix string
}

func (s *SampleService) Label(name string) string {
	return formatName(s.Prefix, name)
}

func formatName(prefix string, name string) string {
	trimmed := strings.TrimSpace(name)
	return prefix + ":" + strings.ToLower(trimmed)
}

func buildGreeting(name string) string {
	return formatName("hello", name)
}

File: test/fixtures/code/sample.rs
use std::fmt;

pub struct SampleService {
    prefix: String,
}

impl SampleService {
    pub fn new(prefix: String) -> Self {
        Self { prefix }
    }

    pub fn label(&self, name: &str) -> String {
        format_name(&self.prefix, name)
    }
}

pub fn format_name(prefix: &str, name: &str) -> String {
    let trimmed = name.trim().to_lowercase();
    format!("{}:{}", prefix, trimmed)
}

pub fn build_greeting(name: &str) -> String {
    format_name("hello", name)
}

fn helper() -> String {
    fmt::format(format_args!("{}", "ok")).to_string()
}
